storage/feed.go

```diff
@@ -195,23 +195,23 @@ type StoreEventListener interface {
 // will run until the Subscription's events channel is closed.
 func ProcessStoreEvents(l StoreEventListener, sub *util.Subscription) {
 	for event := range sub.Events() {
-		switch event := event.(type) {
+		switch specificEvent := event.(type) {
```

Comment:
Seems reasonable. However, fixing go-nyet may be non-trivial, and for now this is the only instance (which we would could whitelist in the makefile, but it's whack-a-mole either way).

I'll add a TODO, though.